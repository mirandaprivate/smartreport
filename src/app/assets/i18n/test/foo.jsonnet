local i18n = import 'src/app/assets/i18n/i18n.libsonnet';

local foo = i18n.Translations(name='foo', pack=i18n.Pack(
    en_us='foo',
    zh_cn='负',
    zh_hk='負',
));

local bar = i18n.Translations(name='bar', pack=i18n.Pack(
    en_us='bar',
    zh_cn='霸',
    zh_hk='霸',
));

local col = i18n.Node(name='col', deps=[foo, bar]);

local block = i18n.Translations(name='block', pack=i18n.Pack(
    en_us='block',
    zh_cn='块',
    zh_hk='塊',
));

local row = i18n.Node(name='row', deps=[block, col]);

local title = i18n.Node(name='title', deps=[row, col]);

title
