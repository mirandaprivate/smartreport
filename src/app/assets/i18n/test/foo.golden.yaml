---
deps:
- deps:
  - name: block
    pack:
    - locale: 1
      text: block
    - locale: 2
      text: 块
    - locale: 3
      text: 塊
  - deps:
    - name: foo
      pack:
      - locale: 1
        text: foo
      - locale: 2
        text: 负
      - locale: 3
        text: 負
    - name: bar
      pack:
      - locale: 1
        text: bar
      - locale: 2
        text: 霸
      - locale: 3
        text: 霸
    name: col
  name: row
- deps:
  - name: foo
    pack:
    - locale: 1
      text: foo
    - locale: 2
      text: 负
    - locale: 3
      text: 負
  - name: bar
    pack:
    - locale: 1
      text: bar
    - locale: 2
      text: 霸
    - locale: 3
      text: 霸
  name: col
name: title
